tracer/src/Datadog.Trace/Telemetry/DTOs/LogMessageData.cs

```diff
@@ -0,0 +1,21 @@
+// <copyright file="LogMessageData.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Vendors.Newtonsoft.Json.Converters;
+
+namespace Datadog.Trace.Telemetry
+{
+    internal class LogMessageData
+    {
+        public string Message { get; set; }
+
+        [Vendors.Newtonsoft.Json.JsonConverter(typeof(StringEnumConverter))]
+        public TelemetryLogLevel Level { get; set; }
+
+        public string Tags { get; set; }
+
+        public string Stack_Trace { get; set; }
```

Comment:
nit :we automatically convert to snake case, no need for it in the C#. + it should be optional
```suggestion
        public string? StackTrace { get; set; }
```

Reply:
Fixed by copy pasting the contents of the entire file over from https://github.com/DataDog/dd-trace-dotnet/pull/3543 in https://github.com/DataDog/dd-trace-dotnet/commit/7ba4bb6bbbcb9992e7874a4625f5a783bbf05002.